### DISCUSSION TITLE
Share in opensource my pictures ?

### DISCUSSION THREAD

**Discussion by ptilou** [ImageMagick/ImageMagick#6012] [Contribute]
Hello evrybody,

Have question, in usenet i'm share many pictures, do you interresting from the project Imagemagick, i shar emy picture ?

It's yes, how to, and where upload my pictures ?

Great 

Philippe

**Comment by fmw42** [comment]
Zip your images and post them to this forum or to some free hosting service (such as dropbox) and then put the URL here.
